sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -4,12 +4,22 @@
 
 ### Features Added
 
+- Fully functional eventhubs Processor.
+- Allow `ProducerClient` and `ConsumerClient` to be created with a connection string without an EntityPath element.
+
 ### Breaking Changes
 
+- Removed the `LoadBalancer` type from the public API surface.
+- `ConsumerClient` and `ProducerClient` objects can no longer be moved or copied.
+- If the connection string provided to `ConsumerClient` or `ProducerClient` contains an EntityPath, then the `EntityPath` 
+parameter to the constructor must match the value provided in the connection string.
+
 ### Bugs Fixed
 
 ### Other Changes
 
+- Several `ostream` insertion operators were added for eventhubs types.
```

Comment:
I always thought that the changelog should include what was changed in a fashion which is meaningful to a consumer (in this case a developer).

When I read "expanded logging coverage" it says to me "We added a bunch more traces", which IMHO isn't worthy of a changelog line (we've certainly never added changelog entries for other cases where we've added traces).

But in this case, there are several ostream insertion operators which were added for types which previously had not been available. That is a customer visible change since they can now use those ostream insertion operators in their code.

And I checked the actual changes from the commit log and there are at least 2 insertion operators which were added for public types (technically the types are in a _detail namespace but they ARE in public headers[1]). Thus the line in the changelog.


[1] These types shouldn't properly be in the public headers - it's an artifact of the AMQP tests that it still exists. But the header is still a public header (for now).